test/integration/cloud-config-aws.ini.template

```diff
@@ -16,6 +16,9 @@ aws_access_key: @ACCESS_KEY
 aws_secret_key: @SECRET_KEY
 security_token: @SECURITY_TOKEN
 aws_region: @REGION
+# aws_cleanup controls whether the environment is cleaned up after tests have completed
+# This only applies to tests that have a cleanup stage
+aws_cleanup: false
```

Comment:
Comment the value out and note that it defaults to true.

```suggestion
# The default value is true when using this template.
# aws_cleanup: false
```

Then update the ansible-test cloud plugin to set the value if it wasn't provided by the user.

After this section:

https://github.com/ansible/ansible/blob/91d326cb68de876cda915683f6b072e5d82bd2c2/test/runner/lib/cloud/aws.py#L100

Add something like:

```python
if 'aws_cleanup' not in ansible_vars:
    ansible_vars['aws_cleanup'] = not self.managed
```

That way it defaults to true for static config (without the setting) and false for dynamic config, while still allowing the setting to be set manually.